src/pages/Scholarships.jsx

```diff
@@ -20,23 +20,26 @@ function ScholarshipsPage() {
   const [scholarships, setScholarships] = useState([]);
   const [loading, setLoading] = useState(true);
   const [error, setError] = useState();
+  const [sortBy, setSortBy] = useState('deadline');
+  const [sortFormat, setSortFormat] = useState('asc');
 
   useEffect(() => {
     // TODO: Create cancellable promises
-    Scholarships.list({ sortField: 'deadline' })
+    Scholarships.list({ sortField: `${sortBy}`, sortDir: `${sortFormat}` })
       .then(setScholarships)
       .catch(setError)
       .finally(() => setLoading(false));
-  }, []);
+  }, [sortBy, sortFormat]);
 
   const classes = useStyles();
 
   return (
     <Container>
+      {console.log(`Currently sorting by: ${sortBy} + ${sortFormat}`)}
```

Comment:
Please delete before merging.

Reply:
Ok, must of forgot to remove this last console.log